File: src/powerup.py
```python
class PowerUp:
    def __init__(self, type, duration):
        self.type = type
        self.duration = duration
        self.active = False

    def activate(self):
        self.active = True

    def deactivate(self):
        self.active = False

def activatePowerup(powerup):
    powerup.activate()

def deactivatePowerup(powerup):
    powerup.deactivate()
```

File: ColorDash/src/main/java/com/colordash/game/PowerUp.java
```java
package com.colordash.game;

public class PowerUp {
    private String type;
    private boolean isActive;

    public PowerUp(String type) {
        this.type = type;
        this.isActive = false;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public boolean isActive() {
        return isActive;
    }

    public void setActive(boolean active) {
        isActive = active;
    }

    public void activatePowerUp() {
        this.isActive = true;
    }

    public void deactivatePowerUp() {
        this.isActive = false;
    }
}
```